### ISSUE TITLE
Changed difficulty flag?

### ISSUE THREAD

**Issue by DevilXD** [KarlitosVII/trilogy-save-editor#78]
Hello,

I'm looking for the "changed difficulty" flag, so that I can reset it. I'm playing ME1 on the Legendary Edition.

Can anyone tell me where to find it? I don't seem to be able to see it in the editor.

**Comment by rgiuffre** [comment]
I found it under Plot, General. But I loaded a LE ME3 save file. I also found it by loading a LE ME2 save, and it's under Plot, Player. But I don't see the option loading a LE ME1 save file.

**Comment by DevilXD** [comment]
Yeah, I'm unable to find it for the ME1 file. Maybe it's under a different name?

**Comment by levicki** [comment]
> I'm looking for the "changed difficulty" flag, so that I can reset it. I'm playing ME1 on the Legendary Edition.

If you don't want "changed difficulty" flag to be set then just don't change difficulty from higher to lower when you play as that's considered cheating.

**Comment by DevilXD** [comment]
> If you don't want "changed difficulty" flag to be set then just don't change difficulty from higher to lower when you play

I know this, I'm not that illiterate as you apparently seem to be thinking.

> that's considered cheating.

I don't think you understand that this whole repo is about a save editor, that (at least in theory) let's you change anything about the save file. That's "cheating" already, by your definition. I was simply asking about a particular part of the save I was editing, and that's it.

**Comment by ThenTech** [comment]
@DevilXD Probably very late, but LE1 doesn't have a "changed difficulty" flag like LE2 and 3. You can edit a save right before the final Saren fight, and change the difficulty in the save editor to Insanity. You'll get the Insanity achievement after defeating him, regardless what the difficulty was at for the rest of the game.

For LE2 and 3, there is the "changed difficulty" toggle under Plot > General (`ChangedDifficulty` bool with id 5616), and possibly also the boolean `Monitor_Difficulty_Level_Player_Cheated` (5170).

**Comment by DevilXD** [comment]
@ThenTech Yeah, very late indeed, I've already finished ME1. I figured it out myself at the end of the game, pretty much. As you said, there's no flag, and you can just edit the difficulty at the end. I only had to turn it down a notch for a single fight, so everything worked out either way.

Thanks for the help =)

**Comment by mirh** [comment]
Correct me if I'm wrong, but didn't OT1 actually have that?
I could swear to remember you had time until the end of the eden prime prologue to decide on a difficulty before the save was locked in for the achievement.

**Comment by DevilXD** [comment]
Yeah, that's what info I was constantly finding on the internet too. Still, I found no flag like that in the editor, and even after playing a part of one mission on a slightly lower difficulty, it still seem to have counted as "no change" at the end of the first game somehow.

Haven't started ME2 yet, so idk how it plays out later.
